apps/website/get-started/updating/angular.md

```diff
@@ -9,10 +9,3 @@ In order to keep up with Clarity, you can use ng update to automatically update
 ng update @angular/core @angular/cli
 ng update @clr/angular
 ```
-
-If you want to update to a prerelease version, you can do so by running:
```

Comment:
I'd keep this and add headers to distinguish between normal upgrades and next upgrades.